client/devdocs/gutenberg-components/example.jsx

```diff
@@ -0,0 +1,77 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import React from 'react';
+import * as components from '@wordpress/components';
+import { withState } from '@wordpress/compose';
+import { getSettings } from '@wordpress/date';
+import { addFilter } from '@wordpress/hooks';
+import { LiveError, LivePreview, LiveProvider } from 'react-live';
+import request from 'superagent';
+import codeBlocks from 'gfm-code-blocks';
+import classnames from 'classnames';
+import { kebabCase } from 'lodash';
+import PropTypes from 'prop-types';
+
+class Example extends React.Component {
+	state = {
+		code: null,
+	};
+
+	componentDidMount() {
+		this.getCode();
+	}
+
+	async getReadme() {
+		const readmeFilePath = `/node_modules/@wordpress/components/src/${
+			this.props.readmeFilePath
+		}/README.md`;
+
+		const { text } = await request.get( '/devdocs/service/content' ).query( {
+			path: readmeFilePath,
+			format: 'markdown',
+		} );
+		return text;
+	}
+
+	async getCode() {
+		const readme = await this.getReadme();
+
+		// Example to render is the first jsx code block that appears in the readme
+		let code = codeBlocks( readme ).find( block => 'jsx' === block.lang ).code;
+
+		// react-live cannot resolve imports in real time, so we get rid of them
+		// (dependencies will be injected via the scope property).
+		code = code.replace( /^.*import.*$/gm, '' );
+
+		code = `${ code } render( <${ this.props.render } /> );`;
+
+		this.setState( { code } );
+	}
+
+	render() {
+		const { code } = this.state;
+		const scope = {
+			...components,
+			withState,
+			getSettings,
+			PropTypes,
+			addFilter,
+		};
+		const className = classnames(
+			'devdocs__gutenberg-components__example',
```

Comment:
Really minor but we generally have a single `__` in the class name.  eg `component__my-long-element-name`. Devdocs is a little odd already, but I wanted to note that. This is non-blocking.

https://github.com/Automattic/wp-calypso/blob/17161ba198e0d18d23d2708e7dcb986816c1dd78/docs/coding-guidelines/css.md#csssass-coding-guidelines